app/assets/javascripts/admin/addon/models/color-scheme.js

```diff
@@ -88,6 +88,28 @@ export default class ColorScheme extends EmberObject {
     return [`"${this.name}": {`, buffer.join(",\n"), "}"].join("\n");
   }
 
+  schemeObject() {
+    const extractColors = (property) =>
+      Object.fromEntries(
+        this.colors.map((color) => [color.get("name"), color.get(property)])
+      );
+    return {
+      dark: extractColors("dark_hex"),
+      light: extractColors("hex"),
+    };
+  }
+
+  /**
+   * @returns a JSON representation of the color scheme.
+   */
+  dump() {
```

Comment:
I named this function dump because in the future there may be a `load` function to restore a `ColorScheme` from a `dump`.